test/live/live.test.ts

```diff
@@ -117,7 +118,7 @@ describe.only('zksync', function () {
 	// }, 60000)
 })
 
-describe('polygon', function () {
+describe.only('polygon', function () {
```

Comment:
idk, but this is how our tests currently work :)

They are not runnable altogether since they are too much broken and it's easier to run them one-by-one when you have .only